### DISCUSSION TITLE
Add Homebrew Support

### DISCUSSION THREAD

**Discussion by leander-hass** [Lennolium/swiftGuard#18] [Feature Requests]
Hi there, really like the concept of this software!
I would like to download it through homebrew. Maybe then your auto-update feature would be easier to implement?

**Comment by Lennolium** [comment]
Thanks for your suggestion! 🚀

I already thought about submitting swiftGuard to Homebrew Cask, but they are requiring/enforcing some points, which right now swiftGuard can not fulfill:

> An app from a code repository that is not notable enough (under 30 forks, 30 watchers, 75 stars).

swiftGuard: 14 forks, 34 watchers and 330 stars.

> App fails with GateKeeper enabled on Homebrew supported macOS versions and platforms (e.g. unsigned apps fail on Macs with Apple Silicon/ARM).

swiftGuard is unsigned, as Apple declined my credit card two times without giving a reason. I also tried with a another AppleID, still had no luck. Furthermore, signing python binaries/.apps is a tortuous job :(

Ref: [brew.sh docs](https://docs.brew.sh/Acceptable-Casks)
